frontend/src/components/Button.js

```diff
@@ -8,6 +8,7 @@ export default styled.button`
   color: var(--neutral-light);
   font-size: 1em;
   border-radius: var(--size-s);
+  margin: 10px;
```

Comment:
Its better to give the parent element padding instead of a margin on the child. This makes your component more reusable. I doubt you want that margin around every button in your app?